### DISCUSSION TITLE
关于借助站赚信用功能，现在大家有看到MAA刷OF-1吗？

### DISCUSSION THREAD

**Discussion by liuyunxinghen** [MaaAssistantArknights/MaaAssistantArknights#3876] [Q&A] [answered]
如题
不知道哪次更新后，发现再也没看到MAA去OF-1赚信用点了。
我平时放假在家的时候就喜欢就看MAA自动完成任务
现在好久没见到了，想问问是大家都有这问题？还是就我遇到的特殊情况。
用的国际版蓝叠模拟器5.10.250.1004 N64
MAA用的开发版，及时更新。

**Comment by ABA2396** [comment, marked as answer]
之前有次更新增加了一天只刷一次的功能，在 `.\config\gui.json` 中的 `Visit.LastCreditFightTaskTime` 有上次刷 OF-1 的日期，如果不是昨天的话看看是不是关卡选择了 `当前/上次`
![image](https://user-images.githubusercontent.com/99072975/222925023-98de7cb1-2e7c-4d48-8e82-dce03ede32c8.png)

**Reply by liuyunxinghen** [reply]
"Visit.LastCreditFightTaskTime": "2023/03/04 00:00:00",
我看了下显示，我3月4日零点刷了，但是我刷理智是在获取信用及购物前面的，之前看关卡记录是显示OF-1的，现在都没看到了。
可以自己改成每次都刷吗？我对于怎么改，改哪里不太了解。

**Reply by ABA2396** [reply]
这个只是日期，时间没存，你可以删除这行然后跑一遍试试会不会去打

**Reply by liuyunxinghen** [reply]
> 这个只是日期，时间没存，你可以删除这行然后跑一遍试试会不会去打

删除后会去打了，谢谢，Thanks♪(･ω･)ﾉ

**Comment by liuyunxinghen** [comment]
设置应该是没有出问题吧，昨晚看现在雷电也没什么问题了，换回雷电了
之前是有正常刷OF-1的，后面更新才没刷过，我看下了要求，应该一直没有冲突过。









流云星痕
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "MaaAssistantArknights/MaaAssistantArknights"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年3月5日(星期天) 凌晨3:25
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [MaaAssistantArknights/MaaAssistantArknights] 关于借助站赚信用功能，现在大家有看到MAA刷OF-1吗？ (Discussion #3876)





 
.\config\gui.json 中的 "Visit.LastCreditFightTaskTime" 有上次刷 OF-1 的日期，如果不是昨天的话看看是不是关卡选择了 当前/上次

**Comment by liuyunxinghen** [comment]
前面用邮箱回复了，照片好像被吞了
![image](https://user-images.githubusercontent.com/42022829/222933665-8da07075-d05e-4c57-821e-ffdf7b0ca107.png)
![image](https://user-images.githubusercontent.com/42022829/222933666-5a40b0f1-a835-41fd-baa5-d2f4738ca808.png)
![image](https://user-images.githubusercontent.com/42022829/222933668-4b310933-c458-4b84-8c92-0d85d52a000d.png)
